src/App.js

```diff
@@ -1,14 +1,88 @@
 import React, { Component } from 'react';
-import logo from './logo.svg';
 import './App.css';
-import { Loop, Stage } from 'react-game-kit';
+import { Loop, Stage, Body, World } from 'react-game-kit';
+import Player from './player';
+import playerSprite from './assets/character-sprite.png';
 
 class App extends Component {
+
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            currentPlayer: {
+                id: 0,
+                name: '',
+                positionX: 100,
+                positionY: 10,
+                speed: 100,
+                directionDegrees: 0,
+                score: 10,
+                lastUpdateTime: 0,
+                sizeMultiple: 2
+            },
+            otherPlayers: []
+        };
+    }
+
+    componentDidMount() {
+        window.addEventListener('keypress', this.handleKeyPress);
+    }
+
+    componentWillUnmount() {
+        window.removeEventListener('keypress', this.handleKeyPress);
+    }
+
+    handleKeyPress = (e) => {
+        const keys = {
+            ENTER: 13,
+            A: 97,  // LEFT
```

Comment:
We should probably map these to actions, not keys. For example, make the property `LEFT` with the comment `A` instead of the other way around.